custom/icds/templates/icds/login.html

```diff
@@ -42,18 +46,16 @@ <h2 class="text-center">Ministry of Women and Child Development</h2>
             <img class="logo" src="{% static 'icds/img/ICDS-CAS_banner.png' %}" alt="CAS Logo" />
         </div>
     </div>
-
     <div class="row">
         <div class="col-sm-2">
-            <img style="height:300px" src="{% static 'icds/img/phone_left.png' %}">
+            <img class="phone" src="{% static 'icds/img/phone_left.png' %}">
         </div>
-
         <div class="col-sm-8" style="padding-top: 40px">
             <h2 class="text-center">Welcome to ICDS-CAS!</h2>
             {% include "login_and_password/two_factor/core/login.html" %}
         </div>
-        <div class="col-lg-2">
-            <img style="height:300px" src="{% static 'icds/img/phone_right.png' %}">
+        <div class="col-sm-2">
```

Comment:
This is the fix for the focus bug, right?

Reply:
I wasn't actually sure what fixed it... by the time I had sanitized the screens it was no longer present. But that sounds right.